internal/cache/serialization.go

```diff
@@ -110,6 +120,14 @@ func dbfromYAML(r io.Reader, destDir string) error {
 
 			for key, val := range bucketContent {
 				if bucketName == userByIDBucketName || bucketName == userByNameBucketName {
+					// Replace CURRENT_USER name by the name of the local user.
+					u, err := user.Current()
+					if err != nil {
+						return err
+					}
+					val = strings.ReplaceAll(val, "CURRENT_USER", u.Username)
+					key = strings.Replace(key, "CURRENT_USER", u.Username, 1)
+
```

Comment:
I didn’t look far, but I feel the logic is incorrect anyway. We don’t care about the CURRENT_USER. It will be root anyway in production for the service, we care about any logged in users having some processes opened.

Reply:
So we should use something like "ACTIVE_USER" and fetch one from `utmp`?

Reply:
The issue from `utmp` is that it’s not that reliable. We did with jibel multiple attempts and it’s not clear.

If you `ls -ld /proc/*`, you will see for the pids the user owning the process. So we can probably loop over that dir